scripts/qa_crowbarsetup.sh

```diff
@@ -1665,6 +1665,24 @@ function custom_configuration()
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value keystone default "['deployment']['keystone']['elements']['keystone-server']" "['cluster:$clusternameservices']"
             fi
+            if [[ $want_ldap ]] ; then
```

Comment:
This block made an already huge function even longer, which really impacts legibility and maintainability.  It's already too late for a -1, but any chance you could submit a new PR to extract it into a separate function?